client/my-sites/patterns/components/search-field/index.tsx

```diff
@@ -41,6 +41,7 @@ export const PatternsSearchField = ( { isCollapsible = false }: PatternsSearchFi
 		<Search
 			additionalClasses={ classNames( { 'is-filled': !! searchTerm } ) }
 			initialValue={ searchTerm }
+			key={ `search-${ category }` }
```

Comment:
It's again here 😅 Hm, could you please provide steps to reproduce the bug (w/o `key`)? Since:
1) I can't reproduce it
2) When we change category - we clear search query in URL, what means that we trigger rerendering of this component due to using `searchTerm` inside here.